### PR TITLE
update adopters

### PR DESCRIPTION
**Pull request by WT-Liu**
Signed-off-by: Wenting Liu <liuwenting@pingcap.com>

Update the list of adopters for TiKV

### Release note
- No release note

### CODE REVIEW

File: docs/adopters.md [hunk 1]
```diff
@@ -5,23 +5,34 @@ category: adopters
...
+|[Shopee](https://en.wikipedia.org/wiki/Shopee)|E-commerce|[English](https://www.pingcap.com/success-stories/tidb-in-shopee/); [Chinese](https://www.pingcap.com/cases-cn/user-case-shopee/)|
+|[LY.com](https://www.crunchbase.com/organization/ly-com)|Travel|[Chinese](https://www.pingcap.com/cases-cn/user-case-tongcheng/)|
+|[Zhuan Zhuan](https://www.crunchbase.com/organization/zhuan-zhuan)|Online Marketplace|[English](https://www.pingcap.com/success-stories/tidb-in-zhuanzhuan/); [Chinese](https://pingcap.com/cases-cn/user-case-zhuanzhuan/)|
+|[Meituan-Dianping](https://en.wikipedia.org/wiki/Meituan-Dianping)|Food Delivery|[English](https://www.pingcap.com/success-stories/tidb-in-meituan-dianping/); [Chinese](https://pingcap.com/cases-cn/user-case-meituan/)|
+|[Ele.me](https://en.wikipedia.org/wiki/Ele.me)|Food Delivery|
+|[Yidian Zixun](https://www.crunchbase.com/organization/yidian-zixun#section-overview)|Media and Entertainment|
+|[KuGou](https://en.wikipedia.org/wiki/KuGou)|Entertainment and Music Streaming|
+|[Meitu](https://en.wikipedia.org/wiki/Meitu)|Photo-editing|
+
+This is a list of TiKV adopters deploying TiKV with TiDB.
```
Comment (dcalvin) [inline docs/adopters.md:22]:
```suggestion
The following are a list of TiKV adopters deploying TiKV with TiDB.
```

**Comment by WT-Liu** [comment]
@dcalvin PTAL again~

**Pull request closed without merging**